src/core_plugins/console/public/src/es.js

```diff
@@ -16,6 +16,10 @@ export function send(method, path, data) {
   var wrappedDfd = $.Deferred();
 
   console.log("Calling " + path);
+  if (data && method == "GET") {
```

Comment:
this needs to be case insensitive, quite often folks do `get /something` and then the body is ignored.